spec/serverSpec.js

```diff
@@ -96,5 +96,61 @@ describe('Dashboard Test', () => {
         });
     });
   });
+
+  describe('Elasticsearch Data', () => {
+    it('3.1) It should have profiles index with correct mappings to types', (done) => {
```

Comment:
Nice use of numbered headings and subheadings. I think I'll incorporate it into my testing as well! 👍